src/networking.c

```diff
@@ -3561,566 +3626,640 @@ void quitCommand(client *c) {
     c->flag.close_after_reply = 1;
 }
 
-void clientCommand(client *c) {
```

Comment:
Could you check what happens to the `parent` ptr for sub command(s) ? We might still need to have a dummy method of `clientCommand` to avoid invalid access of function not defined.

```suggestion
void clientCommand(client *c) {
        addReplySubcommandSyntaxError(c);
}
```

Reply:
yes, this seems to be working. I have added ClientCommand as a function in client.json